docs/apis-clients/working-with-apis-clients.md

```diff
@@ -1,33 +1,22 @@
 ---
 id: working-with-apis-clients
-title: "Working with APIs & Clients"
-sidebar_label: "Working with APIs & Clients"
-description: "Programmatically work with Camunda Platform 8 through APIs & clients"
+title: "Working with APIs & tools"
+sidebar_label: "Working with APIs & tools"
+description: "Interact programmatically with Camunda Platform 8 using official Zeebe client libraries and APIs."
 ---
 
-This section steps through a variety of offered APIs and clients for integration.
+import DocCardList from '@theme/DocCardList';
 
-:::note
-You're permitted to use these web apps and APIs for free with the Free Edition in non-production environments. To use the software in production, [purchase the Camunda Platform Enterprise Edition](https://camunda.com/products/cloud/camunda-cloud-enterprise-contact/). Read more in our [licensing](../reference/licenses.md) documentation.
-:::
-
-## APIs and interacting with other components
+This section steps through two concepts for integration:
```

Comment:
Just an idea:

* The page has now two main concepts: (1) Deploy processes (...) using Zeebe Clients (2) Learn about Camunda Components and their APIs
* We could try to make 2 main concepts more outcome driven. For example:

1. Control your Camunda 8 process automation (e.g., deploy processes, start process instances, ...) using Zeebe Clients
2. Interact with the Camunda 8 ecosystem (e.g., retrieve open User Tasks from Tasklist) using the respective component APIs

Reply:
Adjusted this in commit as well to incorporate the concept of outcome-driven components here.